tests/aggregation_regress/tests.py

```diff
@@ -873,7 +873,7 @@ def test_more_more_more(self):
         )
         self.assertEqual(
             Book.objects.annotate(Count('publisher')).values('publisher').count(),
-            6
```

Comment:
`annotate()` and `values()` should not affect `count()`. Now in both cases we get the same number of books:
```python
Book.objects.values('publisher').annotate(Count('publisher')).count()
Book.objects.annotate(Count('publisher')).values('publisher').count()
```